pkg/ingester/ingester_test.go

```diff
@@ -82,6 +82,36 @@ func matrixToSamples(m model.Matrix) []model.Sample {
 	return samples
 }
 
+// chunksToMatrix converts a slice of chunks into a model.Matrix.
+func chunksToMatrix(chunks []chunk.Chunk) (model.Matrix, error) {
```

Comment:
You could still have this as a method on Chunk, even if its in a _test.go.  But no worries either way.